OpenRA.Mods.Common/Traits/World/ShroudRenderer.cs

```diff
@@ -209,177 +216,71 @@ Edges GetEdges(MPos uv, Func<MPos, bool> isVisible)
 			return info.UseExtendedIndex ? edge ^ ucorner : edge & Edges.AllCorners;
 		}
 
-		public void RenderShroud(WorldRenderer wr, Shroud shroud)
+		void UpdateCells(IEnumerable<CPos> cells)
```

Comment:
If I understand this correctly, this method just marks the given cells as dirty. If so, maybe a better name is in order? Or at least a `<summary>`. We should make more use of those.